sass/blocks/gallery/_gallery.scss

```diff
@@ -0,0 +1,54 @@
+.gallery {
+  &__item {
+    &::after {
+      content: "";
+      display: block;
+      border-top: 1px solid $form-divider;
+    }
+  }
+
+  &__inner {
+    padding: 23px 5.75% 17px 6.25%;
```

Comment:
why is it 5.75% ? and 6.25%?
We need to try to unify all our container spaces and create a mixin, not make it as unique as possible.